cmd/kubelet/app/server.go

```diff
@@ -152,10 +152,7 @@ periodically for updates. The monitoring period is 20s by default and is configu
 via a flag.
 
 HTTP endpoint: HTTP endpoint passed as a parameter on the command line. This endpoint
-is checked every 20 seconds (also configurable with a flag).
-
-HTTP server: The kubelet can also listen for HTTP and respond to a simple API
-(underspec'd currently) to submit a new manifest.`,
+is checked every 20 seconds (also configurable with a flag).`,
```

Comment:
```suggestion
is checked every 20 seconds (also configurable with a flag).`.
```

Reply:
I'm a little confused, this location `,` should exist as a separator in golang syntax

Reply:
you are correct, my mistake